UnitsNet/GeneratedCode/Quantities/Acceleration.NetFramework.g.cs

```diff
@@ -836,6 +836,16 @@ public string ToString([CanBeNull] IFormatProvider provider, [NotNull] string fo
         /// </summary>
         /// <param name="format">The format string.</param>
         /// <returns>The string representation.</returns>
+        /// <remarks>
+        /// The valid format strings are as follows:
+        /// "g": The quantity formatted as the value with 2 significant digits after the radix, and abbreviation. For example: 1.23 m
+        /// "a": The default abbreviation for the quantity's unit.
+        /// "a0", "a1", "a2", etc.: The abbreviation with index where multiple abbreviations exist. "a0" is the same as "a" or the default abbreviation.
+        /// "v": The quantity's value outputted as a string using the default representation.
+        /// "u": The quantity's unit
+        /// "q": The quantity's name.
+        /// "s1", "s4", etc.: The quantity formatted as the value with n significant digits after the radix, and abbreviation. For example, "s4" would give: 1.2345 m
```

Comment:
`"s1", "s2", ..., "sN": The value with N significant digits after the radix followed by the unit abbreviation. For example, "s4" would return "1.2345 m" if <see cref="Value" /> is 1.2345678. Trailing zeros are omitted.`